terraform/development/dynamodb.tf

```diff
@@ -4,6 +4,8 @@ resource "aws_dynamodb_table" "transactions_table" {
     read_capacity         = 10
     write_capacity        = 10
     hash_key              = "id"
+    range_key             = "target_id"
```

Comment:
Hi Edward, I think this should be as follows:   
  hash_key       = "targetId"
  range_key      = "id"

Reply:
My bad, Done